### ISSUE TITLE
Displaying vertical cards instead of horizontals

### ISSUE THREAD

**Issue by ugenc** [dabeng/OrgChart#567]
Hi Xuebin Dong, it is an awesome project to play around. But I wonder how to display my cells in the last level not side-by-side, but head-to-head. 

Example; (displaying cards as in the yellow area)

![Artboard 1-100](https://user-images.githubusercontent.com/219390/82324167-dfd21580-99e1-11ea-8d47-d87b55df1673.jpg)

thank you for your help :)

**Comment by VinceGee** [comment]
I believe what you are looking for is simple: for the highlighted portion:

`{ 'name': 'Hei Hei', 'title': 'senior engineer',
              'children': [
                { 'name': 'Dan Zai', 'title': 'UE engineer',
                  'children': [
                    { 'name': 'Er Dan Zai', 'title': 'engineer',
                    }
                  ]
                }
              ]
            }`

**Comment by ugenc** [comment]
Yeah, thank you for this opinion. That's one solution. And, there is an vertical-depth option. it also works.

**Comment by mahindar-s4** [comment]
Hey @ugenc , @VinceGee @dabeng 
Can you guys help me on how to drag and drop a node from one chart to another chart within same page.
Also, how to create sibling nodes for a parent node?

Appreciate your help
